scheduler/all-base.yaml

```diff
@@ -108,6 +108,7 @@ services:
     image: "${KAFKA_IMAGE_AND_TAG}"
     environment:
     - KAFKA_CFG_SOCKET_REQUEST_MAX_BYTES=1000000000
+    - KAFKA_CFG_MESSAGE_MAX_BYTES=1000000000
```

Comment:
:thought_balloon: Ideally these would be controlled by the same env var, so that users can easily configure this.  For example, one might want to simulate a production system that imposes a lower limit, which is common for hosted Kafka solutions.

Reply:
Good point - let me change that